classes/03class/exercises/c03-docker01/Faria-Marcio/Dockerfile

```diff
@@ -0,0 +1,13 @@
+#Alpine docker image with Terraform
+
+FROM alpine:latest
+
+MAINTAINER Marcio Faria <marciodefaria@gmail.com>
+
+RUN apk add wget unzip \
+	&& wget https://releases.hashicorp.com/terraform/0.12.20/terraform_0.12.20_linux_amd64.zip \

```

Comment:
can you use the `--quiet` option to avoid such a long file for the c03-docker01.txt file?